src/libstd/ffi/os_str.rs

```diff
@@ -188,6 +188,16 @@ impl OsString {
     /// in the given `OsString`.
     ///
     /// The collection may reserve more space to avoid frequent reallocations.
+    ///
+    /// # Examples
+    ///
+    /// ```rust
```

Comment:
I thought that the RFC stated to not use them more strongly, but apparently it does not.